.cargo/config.toml

```diff
@@ -12,8 +12,6 @@ rustflags = [
     "-Clink-arg=-sALLOW_TABLE_GROWTH=1",
     "-Clink-arg=--js-library=js/embed/soljson_interface.js",
     "-Clink-arg=--pre-js=js/embed/pre.js",
-    "-Clink-arg=-sNODEJS_CATCH_EXIT=0",
-    "-Clink-arg=-sDISABLE_EXCEPTION_CATCHING=0",
     "-Clink-arg=-sSTACK_SIZE=128kb",
-    "-Copt-level=3"
+    "-Clink-arg=-sNODEJS_CATCH_EXIT=0"
```

Comment:
Are you removing the opt level on purpose?

Reply:
I see in logs that compiling in release mode adds this option automatically

Reply:
Thanks for the clarification.